src/service/resources-impl.js

```diff
@@ -331,16 +382,14 @@ export class ResourcesImpl {
           r,
           /* checkForDupes */ true,
           /* scheduleWhenBuilt */ false,
-          /* force */ true
+          /* ignoreQuota */ true
         );
         dev().fine(TAG_, 'force build:', r.debugid);
       }
 
       // TODO(willchou): Risk of long task due to long microtask queue?
       return r.whenBuilt().then(() => {
-        const wasIntersecting = r.isInViewport();
         let isDisplayed = this.measureResource_(r, clientRect);
-
         if (wasIntersecting && !isIntersecting) {
```

Comment:
in which situation would `isDisplayed` and `isIntersecting` have different values.

Reply:
`isDisplayed()` reflects size of the bounding rectangle. It should probably be renamed. Happy to do that separately.

Reply:
ah, like hidden vs. visible. the name isn't bad, but when it comes to viewport detection "displayed" is definitely overloaded